BrowserKit/Sources/Logger/CrashManager.swift

```diff
@@ -78,6 +78,8 @@ public class DefaultCrashManager: CrashManager {
                 }
                 return crumb
             }
+            // Turn Sentry breadcrumbs off since we have our own log swizzling
+            options.integrations = options.integrations?.filter { $0 != "SentryAutoBreadcrumbTrackingIntegration" }
```

Comment:
nit: Not a blocker but I think you can achieve the same with this without relying on the implementation detail of the string it converts to. 
options.enableAutoBreadcrumbTracking = false

Reply:
Ah, much better! Thanks!